proto/ocsf/ocsf_ext/finding_info/v1/finding_info.proto

```diff
@@ -47,3 +47,24 @@ message DataSource {
     PurlFindingLocationData purl_finding_location_data = 4;
   }
 }
+
+
+// annotations created by the DataEnricher
+message Enrichment {
+  enum EnrichmentType {
+    ENRICHMENT_TYPE_UNSPECIFIED = 0;
+    ENRICHMENT_TYPE_CORRELATION = 1;
+    ENRICHMENT_TYPE_DUPLICATION = 2;
+  }
+
+  message CorrelationEnrichment {
+    repeated string findings = 1;
+  }
+
+  EnrichmentType enrichment_type = 1;
+
+  oneof enrichment {
+    bool duplicate = 2;
```

Comment:
I think that it'd be better to use messages here so that they are open for changes directly

For example, we might want to add some extra fields like num_of_duplicates to the duplication enrichment and so

By having message there's no problem adding such property :)